bids2openminds/main.py

```diff
@@ -19,17 +19,17 @@ def create_techniques(layout_df):
         if not (pd.isna(suffix) or (suffix in not_techniques_index)):
             techniques.extend(bids2openminds_instance(suffix, "MAP_2_TECHNIQUES"))
 
-    return techniques
+    return techniques or None
 
 
 def create_approaches(layout_df):
     datatypes = layout_df["datatype"].unique().tolist()
-    approaches = []
+    approaches = set([])
     for datatype in datatypes:
         if not (pd.isna(datatype)):
-            approaches.extend(bids2openminds_instance(datatype, "MAP_2_EXPERIMENTAL_APPROACHES"))
+            approaches.update(bids2openminds_instance(datatype, "MAP_2_EXPERIMENTAL_APPROACHES"))
 
-    return approaches
+    return list(approaches) or None
 
 
 def dataset_version_create(bids_layout, dataset_description, layout_df, studied_specimens, file_repository):
```

Comment:
could we change this in this PR already to `def create_dataset_version(...)` for consistency (cf. `def create_approaches(...)`)

Reply:
leave it to the next PR if too much refactoring is required

Reply:
Agreed but again I think we should do it in another pull request.